packages/dapp/src/redux/reducers/ui.ts

```diff
@@ -11,7 +11,7 @@ export function ui(state: Map<string, any> = Map<string, any>(), action: AnyActi
   }
 }
 
-export function useGraphQL(state: boolean = true, action: AnyAction): boolean {
+export function useGraphQL(state: boolean = false, action: AnyAction): boolean {
```

Comment:
Minor suggestion, but I was thinking that we might want a dapp `constants.ts` file that stores stuff like the default value for `useGraphQL`, so they're easier to find.

Reply:
good idea but I'll hold off for now. long term we'll probably want useGraphQL to be in local storage anyways?